glean-core/ios/Glean/Config/Configuration.swift

```diff
@@ -9,7 +9,6 @@ public struct Configuration {
     let userAgent: String
     let connectionTimeout: Int
```

Comment:
We have now removed this from the Kotlin implementation. Both the connection and the read timeouts are implementation details of the uploader class: I'd recommend removing it from here as well (and from the docs).